.circleci/config.yml

```diff
@@ -200,6 +200,15 @@ commands:
     description: >-
       skip the test job is project variabe OVERRIDE_TEST is set with TRUE
     steps:
+      - run:
+          name: skip test in master
+          command: |
+              notskipflag=$(echo "${NOT_SKIPTESTINMASTER}" |  tr '[:upper:]' '[:lower:]')
```

Comment:
1. Where is `NOT_SKIPTESTINMASTER` set?
1. Why are we setting a `NOT` flag instead of defaulting to always running tests and forcing the preceding job (which I assume is a step that merges from the `develop` branch and recognizes that all tests have passed) to set the "skip" flag?

Reply:
NOT_SKIPTESTINMASTER is set in circleci project environment variable
the reason we use NOT flag is to skip the test in master by default, that means when the flag is not set, we will skip the test as we already do the test in develop branch